src/frontend/js/modules/i18n/i18n.service.js

```diff
@@ -25,15 +25,20 @@ angular.module('esn.i18n')
         });
     }
 
-    function translate(text) {
+    /**
+     * Translate a string
+     * @param {String} text The text that needs to be translated
+     * @param {Object} interpolateParams An object hash for dynamic values
+     * @param {Boolean} ignoreSanitizeStrategy Ignore sanitize strategy. Default is false and sanitize strategy is "escape".
+     */
+    // TODO: Write tests for this (https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/issues/83)
+    function translate(text, interpolateParams = {}, ignoreSanitizeStrategy = false) {
       if (!text || text instanceof EsnI18nString) {
```

Comment:
If text = null or undefined, it should return an empty string, right?

Reply:
Not sure since this is an old logic. Maybe this is intended at the beginning.
Bc, there is no bug for this so far. I will keep it like this.